<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Line Analyzer</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen_CSV"/>
   <addaction name="actionSave_Selection"/>
   <addaction name="actionReset_Windows"/>
   <addaction name="actionDelete_Selection"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
  </widget>
  <action name="actionSave_Selection">
   <property name="icon">
    <iconset resource="trendline_analiz_qt.qrc">
     <normaloff>:/ico/ico/save.png</normaloff>:/ico/ico/save.png</iconset>
   </property>
   <property name="text">
    <string>Save Selection</string>
   </property>
   <property name="toolTip">
    <string>Save Graphics Selection</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionOpen_CSV">
   <property name="icon">
    <iconset resource="trendline_analiz_qt.qrc">
     <normaloff>:/ico/ico/open.png</normaloff>:/ico/ico/open.png</iconset>
   </property>
   <property name="text">
    <string>Open CSV</string>
   </property>
   <property name="toolTip">
    <string>Opens a CSV File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionReset_Windows">
   <property name="icon">
    <iconset resource="trendline_analiz_qt.qrc">
     <normaloff>:/ico/ico/refresh.png</normaloff>:/ico/ico/refresh.png</iconset>
   </property>
   <property name="text">
    <string>Reset Windows</string>
   </property>
   <property name="toolTip">
    <string>Reset Windows</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionDelete_Selection">
   <property name="icon">
    <iconset resource="trendline_analiz_qt.qrc">
     <normaloff>:/ico/ico/delete.png</normaloff>:/ico/ico/delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete Selection</string>
   </property>
   <property name="toolTip">
    <string>Delete Selection</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="trendline_analiz_qt.qrc">
     <normaloff>:/ico/ico/undo.png</normaloff>:/ico/ico/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="toolTip">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="trendline_analiz_qt.qrc">
     <normaloff>:/ico/ico/redo.png</normaloff>:/ico/ico/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="toolTip">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="trendline_analiz_qt.qrc"/>
 </resources>
 <connections/>
</ui>
